gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -135,8 +144,11 @@ typedef struct _GstTensorFilterProperties
   int latency; /**< The average latency over the recent 10 inferences in microseconds */
   int throughput; /**< The average throughput in the number of outputs per second */
   int invoke_dynamic; /**< True for supporting invoke with flexible output. */
-
+  int invoke_async; /**< Use this when the sub-plugin receives an input and generates multiple outputs asynchronously. */
```

Comment:
1. Don't insert a new property. Append it.

2. This is "True for supporting invoke asynchronously". GstTensorFilterProperty is a const object that a framework defines. Wheter to invoke async or not should be determined by the user. And using "async" mode should never be madatory for users.

Reply:
Okay, thank you.